.gitattributes

```diff
@@ -1 +0,0 @@
-*.pb*.go -diff linguist-generated=true
```

Comment:
why is this removed?

Reply:
We have a lot of proto plugins, I think it's useful to see the diffs when they change